semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -194,18 +195,29 @@ object Javacp {
             method.signature.params
           }
 
-        val parameterSymbols = params.zipWithIndex.map {
+        val parameterSymbols: List[String] = params.zipWithIndex.map {
           case (param: JavaTypeSignature, i) =>
             val paramName = {
               if (method.node.parameters == null) "param" + i
               else method.node.parameters.get(i).name
             }
             val paramSymbol = Symbols.Global(methodSymbol, d.Parameter(paramName))
+            val isRepeatedType = method.node.access.hasFlag(o.ACC_VARARGS) && i == params.length - 1
```

Comment:
This was hard to find, but the `ACC_VARARGS` flag applies to the whole method.

A parameter is then a vararg if it's the last of a method with `ACC_VARARGS`.

Reply:
Nice find! SemanticDB exists so that tool authors don't have to bother themselves with stuff like that.

Reply:
Making vararg a property of the method makes sense to me semantically. One awkward artifact of s.ByNameType is that it's possible to encode invalid states
```scala
def foo(a: *A, b: *B)
```
On the other hand, by-name types makes sense for syntax trees where Defn.Def has no attributes/modifiers to encode varargs 🤔